python/ray/_private/log_monitor.py

```diff
@@ -24,8 +24,8 @@
 # entry/init points.
 logger = logging.getLogger(__name__)
 
-# The groups are worker id, job id, and pid.
-JOB_LOG_PATTERN = re.compile(".*worker-([0-9a-f]+)-([0-9a-f]+)-(\d+)")
+# The groups are job id, and pid.
+JOB_LOG_PATTERN = re.compile(".*worker.*-([0-9a-f]+)-(\d+)")
```

Comment:
The first group was never used later and not printed out by java

Reply:
What's the first group for? Why not just making java create the same file format?

Reply:
it is for worker id, it's just not used at all.